src/components/Gui.js

```diff
@@ -29,7 +25,6 @@ var Gui = (function () {
         callback(url);
     };
     var showWeather = function (data) {
-        console.log(data);
         var temp = Math.floor(data.main.temp);
         var currentWeather = data.weather[0].main;
         console.info(currentWeather);
```

Comment:
Dont print anything to the console. It affects the performance of your app.

Reply:
done